net/http/src/THttpServer.cxx

```diff
@@ -924,18 +828,19 @@ void THttpServer::ProcessRequest(THttpCallArg *arg)
          // if accepted, reply with connection id, which must be used in the following communications
          arg->SetMethod("WS_CONNECT");
 
-         if (handler && handler->HandleWS(arg)) {
-            arg->SetMethod("WS_READY");
-
-            TLongPollEngine *handle = new TLongPollEngine("longpoll", arg->fPathName.Data());
+         THttpLongPollEngine *handle = new THttpLongPollEngine("longpoll", arg->fPathName.Data());
+         arg->SetWSId(handle->GetId());
 
-            arg->SetWSId(handle->GetId());
+         if (handler->HandleWS(arg)) {
+            arg->SetMethod("WS_READY");
```

Comment:
Can we use enums for those? I'm terribly scared of having random spelling mistakes in these strings... I know you need to send a string in the end, but then we can have an array of strings, and `(int)method` (i.e. the enum constant converted to an int) gives you the index in the string array.

Reply:
On the first place THttpCallArg is class, which represent HTTP request arguments. HTTP header has only string fields, "Method" is very first field and can be "GET", "HEAD", "POST" and few others. I misuse this method name for websockets handling. 
In my mind method should remain string

Reply:
> In my mind method should remain string

Why is that?